src/twisted/internet/endpoints.py

```diff
@@ -2419,3 +2427,38 @@ def parseStreamClient(reactor, *args, **kwargs):
         @rtype: L{IStreamClientEndpoint}
         """
         return _parseClientTLS(reactor, *args, **kwargs)
+
+
+@implementer(IPlugin, IStreamServerEndpointStringParser)
+class _TLSServerEndpointParser:
```

Comment:
This needs to be documented in `docs/core/howto/endpoints.rst`

Reply:
yep, for sure, this is the most important missing piece